tasks/main.yml

```diff
@@ -6,6 +6,15 @@
     - "../vars/empty.yml"
   tags: [always]
 
+#
```

Comment:
Some of the defaults are now different between the versions... so we need a way to override the defaults/main.yml file depending on the version.  There are many ways to do this, but I figured it was best to stick with the same way that it was done for the OS.